### ISSUE TITLE
Support Gigabyte B550M AORUS ELITE

### ISSUE THREAD

**Issue by s-kostyuk** [t-8ch/linux-gigabyte-wmi-driver#18]
B550M AORUS ELITE (v1.3) uses the IT8689 super I/O chip:

```
Some Super I/O chips contain embedded sensors. We have to write to
standard I/O ports to probe them. This is usually safe.
Do you want to scan for Super I/O sensors? (YES/no):
Probing for Super-I/O at 0x2e/0x2f
Trying family `ITE'...                                      Yes
Found unknown chip with ID 0x8689
```

The WMI driver loads and populates the hwmon interface with `force-load=1`:

```
srgk@serhiis-dlbox:~/it87$ sudo modprobe gigabyte-wmi force_load=1
srgk@serhiis-dlbox:~/it87$ sensors
k10temp-pci-00c3
Adapter: PCI adapter
Tctl:         +27.4°C
Tccd1:        +26.2°C

acpitz-acpi-0
Adapter: ACPI interface
temp1:        +16.8°C  (crit = +20.8°C)
temp2:        +16.8°C  (crit = +20.8°C)

gigabyte_wmi-virtual-0
Adapter: Virtual device
temp1:        +25.0°C
temp2:        +35.0°C
temp3:        +27.0°C
temp4:        +23.0°C
temp5:        +26.0°C
temp6:        +35.0°C

nvme-pci-0100
Adapter: PCI adapter
Composite:    +33.9°C  (low  = -273.1°C, high = +84.8°C)
                       (crit = +84.8°C)
Sensor 1:     +33.9°C  (low  = -273.1°C, high = +65261.8°C)
Sensor 2:     +36.9°C  (low  = -273.1°C, high = +65261.8°C)
```

More DMI information:

```
srgk@serhiis-dlbox:~/it87$ sudo grep . /sys/class/dmi/id/* 2> /dev/null
/sys/class/dmi/id/bios_date:11/14/2022
/sys/class/dmi/id/bios_release:5.17
/sys/class/dmi/id/bios_vendor:American Megatrends International, LLC.
/sys/class/dmi/id/bios_version:FB
/sys/class/dmi/id/board_asset_tag:Default string
/sys/class/dmi/id/board_name:B550M AORUS ELITE
/sys/class/dmi/id/board_serial:Default string
/sys/class/dmi/id/board_vendor:Gigabyte Technology Co., Ltd.
/sys/class/dmi/id/board_version:x.x
/sys/class/dmi/id/chassis_asset_tag:Default string
/sys/class/dmi/id/chassis_serial:Default string
/sys/class/dmi/id/chassis_type:3
/sys/class/dmi/id/chassis_vendor:Default string
/sys/class/dmi/id/chassis_version:Default string
/sys/class/dmi/id/modalias:dmi:bvnAmericanMegatrendsInternational,LLC.:bvrFB:bd11/14/2022:br5.17:svnGigabyteTechnologyCo.,Ltd.:pnB550MAORUSELITE:pvr-CF:rvnGigabyteTechnologyCo.,Ltd.:rnB550MAORUSELITE:rvrx.x:cvnDefaultstring:ct3:cvrDefaultstring:skuDefaultstring:
/sys/class/dmi/id/product_family:B550 MB
/sys/class/dmi/id/product_name:B550M AORUS ELITE
/sys/class/dmi/id/product_serial:Default string
/sys/class/dmi/id/product_sku:Default string
/sys/class/dmi/id/product_uuid:03560274-043c-0564-f806-b90700080009
/sys/class/dmi/id/product_version:-CF
/sys/class/dmi/id/sys_vendor:Gigabyte Technology Co., Ltd.
/sys/class/dmi/id/uevent:MODALIAS=dmi:bvnAmericanMegatrendsInternational,LLC.:bvrFB:bd11/14/2022:br5.17:svnGigabyteTechnologyCo.,Ltd.:pnB550MAORUSELITE:pvr-CF:rvnGigabyteTechnologyCo.,Ltd.:rnB550MAORUSELITE:rvrx.x:cvnDefaultstring:ct3:cvrDefaultstring:skuDefaultstring:
```

**Comment by s-kostyuk** [comment]
Talking about fans - I managed to read the RPM values using the `frankcrawford/it87` DKMS driver and `acpi_enforce_resources=lax` in the kernel boot parameters. I have no idea about the fan readout support via WMI. Probably it is supported, but not documented, since I can control the fans via a proprietary tool from Gigabyte on Windows.

**Comment by t-8ch** [comment]
Thanks for the report!

I won't do anything about it right now, as beginning with 6.5 the whole allowlist and `force_load` will completely go away.
So `force_load=1` will effectively will be the default.
Unfortunately in the meantime we can't add new hardware IDs, so it will be a bit annoying during the transition phase.

**Comment by t-8ch** [comment]
Unfortunately anything involving `acpi_enforce_resources=lax` is a complete no-go for upstream defaults.

**Comment by s-kostyuk** [comment]
> Unfortunately in the meantime we can't add new hardware IDs, so it will be a bit annoying during the transition phase.

It is fine. I am using the old Ubuntu 5.19 kernel anyway, so will resort to the `force_load=1` workaround in `modprobe.d`.

> Unfortunately anything involving `acpi_enforce_resources=lax` is a complete no-go for upstream defaults.

I understand that. What I am saying is that this option is available for experimentation. Whether we do any experiments with it is a separate question.

Thank you.

**Comment by t-8ch** [comment]
If someone figures out how to change RPM values via WMI this would absolutely be a candidate for this driver.
I was not able to do so far.

**Comment by boroda24** [comment]
Hello guys, I am using a gigabyte b850m aorus elite ice board, on the ite it8696e controller I mistakenly downloaded a file from the gigabyte website (ITE inboxI2C predefine up v1.0.0.9) and installed it, after which the rgb backlight stopped working (turning on the control does not work). it doesn't work or show up in any program) and only after that I decided to look at the file, and as it turned out, it was a file for the gigabyte b650m aorus elite ice board.I ask you, who have such a board or such a controller, save the firmware. I do not know what exactly it is called, it is responsible for rgb illumination.I'm sorry for the mistakes. I communicate through an interpreter.

**Comment by IGGOR1962** [comment]
Is Linux supported?
